### DISCUSSION TITLE
Any way to use the new function_calling features in the OpenAI API via Guidance?

### DISCUSSION THREAD

**Discussion by TheSylvester** [guidance-ai/guidance#234] [Q&A]
This is going to be a game-changer for those linking custom tools into their LLM usage logic.
I could not determine a way to somehow send that "functions" array through guidance.  I tried passing it in through 

`guidance("my prompt", functions=[{...},{...}], llm)`

 but it didn't work  Has anyone had any success?

**Comment by slundberg** [comment]
Support will be landing for this soon, but not there yet.

**Reply by slundberg** [reply]
Check out #239 for more discussion.
